frontend/src/screens/History/index.js

```diff
@@ -4,16 +4,30 @@ import useCustomContext from '../../customHooks/Hook';
 import ReactPlayer from 'react-player';
 import styles from './history.module.css';
 import { CLEAR_HISTORY, REMOVE_FROM_HISTORY } from '../../constants/type';
+import axios from 'axios'
+import { toast } from 'react-toastify';
 
 const History = () => {
     const { state: { history }, dispatch } = useCustomContext();
 
-    const deleteFromHistory = (id) => {
-        dispatch({ type: REMOVE_FROM_HISTORY, payload: id })
+    const deleteFromHistory = async (id) => {
+        try {
+            const { data: { historyvideos } } = await axios.delete(`${process.env.REACT_APP_SERVER_URL}/api/history/${id}`, { headers: { 'x-auth-token': localStorage.getItem('token') } })
```

Comment:
historyvideos are coming from backend itself as a field. So changing this would means to change from models to frontend code which is more time consuming for now. Will do it next version
I'll keep the naming suggestion and try not to repeat the same mistake again.
Thanks :)